### ISSUE TITLE
[BUG] Invalid BaseTexture Error

### ISSUE THREAD

**Issue by Sinled** [sdenec/tidy5e-sheet#748]
**Module Version:** v0.8.35

**Describe the bug**
`tidy5e-sheet/images/exhaustion.svg` wont render on token, and other probably too if they are rendered on canvas

**To Reproduce**
1. Open In Firefox
2. Add exhaustion level
3. Check token
4. See Warning icon instead loading
5. Check console, there will be 'Invalid BaseTexture'

**Expected behavior**
Correct icon should render on token

**Screenshots**
<img width="178" alt="Screenshot 2023-05-02 at 13 23 06" src="https://user-images.githubusercontent.com/1402135/235642497-d49e594c-67be-44b4-ba54-4390c505f57f.png">

<details>
<summary>Error log</summary>
```
Error: Invalid BaseTexture modules/tidy5e-sheet/images/exhaustion.svg
    loadTexture foundry.js:26082
    _drawEffect foundry.js:46565
    drawEffects foundry.js:46507
    _draw foundry.js:46180
    draw foundry.js:25199
    call_wrapped libWrapper-wrapper.js:507
    tokenDraw monks-active-tiles.js:1513
    call_wrapper libWrapper-wrapper.js:616
    draw#0 libWrapper-wrapper.js:189
    promises foundry.js:30825
    map commons.js:3332
    _draw foundry.js:30823
    _draw foundry.js:37066
    draw foundry.js:30349
    draw foundry.js:26151
    draw foundry.js:39270
    draw foundry.js:23673
    view foundry.js:20901
    call_wrapped libWrapper-wrapper.js:507
    sceneView monks-scene-navigation.js:451
    call_wrapper libWrapper-wrapper.js:616
    sceneView monks-tokenbar.js:176
    call_wrapper libWrapper-wrapper.js:616
    view#0 libWrapper-wrapper.js:189
    initializeCanvas foundry.js:8095
    setupGame foundry.js:7929
    _initializeGameView foundry.js:8999
    _initializeView foundry.js:8975
    initialize foundry.js:7825
    call_wrapped libWrapper-wrapper.js:507
    libWrapperInit libWrapper-api.js:803
    initialize#0 libWrapper-wrapper.js:187
    <anonymous> foundry.js:79693
    async* foundry.js:79675
foundry.js:26089:13
    loadTexture foundry.js:26089
    _drawEffect foundry.js:46565
    drawEffects foundry.js:46507
    _draw foundry.js:46180
    draw foundry.js:25199
    call_wrapped libWrapper-wrapper.js:507
    tokenDraw monks-active-tiles.js:1513
    call_wrapper libWrapper-wrapper.js:616
    draw#0 libWrapper-wrapper.js:189
    promises foundry.js:30825
    map commons.js:3332
    _draw foundry.js:30823
    _draw foundry.js:37066
    draw foundry.js:30349
    draw foundry.js:26151
    draw foundry.js:39270
    draw foundry.js:23673
    view foundry.js:20901
    call_wrapped libWrapper-wrapper.js:507
    sceneView monks-scene-navigation.js:451
    call_wrapper libWrapper-wrapper.js:616
    sceneView monks-tokenbar.js:176
    call_wrapper libWrapper-wrapper.js:616
    view#0 libWrapper-wrapper.js:189
    initializeCanvas foundry.js:8095
    setupGame foundry.js:7929
    _initializeGameView foundry.js:8999
    _initializeView foundry.js:8975
    initialize foundry.js:7825
    call_wrapped libWrapper-wrapper.js:507
    libWrapperInit libWrapper-api.js:803
    initialize#0 libWrapper-wrapper.js:187
    <anonymous> foundry.js:79693
    <anonymous> foundry.js:79675
```
</details>

**Browser:**
 - Firefox

**Foundry Version:** 
- 10.291

**Game System:** 
- 5e v2.1.5

**Additional context**

Problem can be solved if you add correct width and height attributes to corresponding svg files
in case of `exhaustion.svg` it works for me, when i add `width="14" height="14" `

**Comment by p4535992** [comment]
@Sinled uninstall the module and install with this json manifest:

https://github.com/sdenec/tidy5e-sheet/releases/download/0.8.40/module.json

**Comment by Sinled** [comment]
@p4535992 no, sadly it doesn't help

**Comment by p4535992** [comment]
you have the image on the browser cache. Clean up the cache on the browser.

**Comment by Sinled** [comment]
no, i've checked with explicitly disabled cache after plugin update, 
with cache enabled i've received my manually edited image, that worked fine

why do you think it should work? It seems there are no changes in file https://raw.githubusercontent.com/sdenec/tidy5e-sheet/0.8.40/src/images/exhaustion.svg 

Or were there  some other fix in between 0.8.35 and 0.8.40 that should help?
